dev/install-common-deps.sh

```diff
@@ -28,7 +28,8 @@ conda create -q -n test-environment python=3.6
 source activate test-environment
 
 python --version
-pip install --upgrade pip==19.3.1
+pip install --upgrade pip
+pip --version
```

Comment:
Nit:
Shall we use 
```
echo "Upgrade pip to version: `pip --version`"
```
to make it more clear ?

Reply:
@WeichenXu123 Thanks for the comment. I think `pip install --upgrade pip` prints out similar message. What do you think?

```
++ pip install --upgrade pip
Requirement already satisfied: pip in /usr/share/miniconda/envs/test-environment/lib/python3.6/site-packages (20.3.3)
Collecting pip
  Downloading pip-21.0.1-py3-none-any.whl (1.5 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 20.3.3
    Uninstalling pip-20.3.3:
      Successfully uninstalled pip-20.3.3
Successfully installed pip-21.0.1
++ pip --version
pip 21.0.1 from /usr/share/miniconda/envs/test-environment/lib/python3.6/site-packages/pip (python 3.6)
```

Reply:
then it's fine